platform/viewer/src/connectedComponents/ViewerLocalFileData.js

```diff
@@ -41,6 +41,10 @@ class ViewerLocalFileData extends Component {
         studyMetadata.createDisplaySets(sopClassHandlerModules);
       studyMetadata.setDisplaySets(study.displaySets);
 
+      studyMetadata.forEachDisplaySet(displayset => {
+        displayset.localFile = true;
+        displayset.studyIndex = studyIndex;
```

Comment:
The array is almost always going to have < 5 entries. 99% of the time it will have only one entry, so the find will be basically instantaneous anyway. Cleaner to have one less property.

Reply:
ok done